docs/cni-plugin.md

```diff
@@ -35,16 +35,33 @@ Another example with a trunk port and jumbo frames:
 }
 ```
 
+Another example with QinQ configured:
+
+```json
+{
+    "name": "myqinqnet",
+    "type": "ovs",
+    "bridge": "mynet2",
+    "vlan": 1000,
+    "cvlan": [ { "id" : 24 }, { "minID" : 100, "maxID" : 102 } ]
+}
+```
+
 ## Network Configuration Reference
 
 * `name` (string, required): the name of the network.
 * `type` (string, required): "ovs".
 * `bridge` (string, required): name of the bridge to use.
 * `vlan` (integer, optional): VLAN ID of attached port. Trunk port if not
-   specified.
+  specified.
+* `vlanMode` (string, optional): Vlan mode of atached port. One of access, 
+dot1q-tunnel, native-tagged, native-untagged, or trunk. Access if not specified.
```

Comment:
What do you mean by `Access if not specified`?
At the moment, only `trunk`, `access` and `dot1q-tunnel` are supported, right? (unsure about the native part)
Some are implicit, like `access` and `trunk`, so you can express the logic here.

I think that for this PR, you just need to support setting it to `dot1q-tunnel`, the others can be added in a separate PR.

Reply:
Indeed, thanks. I rewrote the descriptions, hopefully it is more clear now.